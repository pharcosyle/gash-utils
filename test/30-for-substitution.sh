for file in `echo foo.o bar.o`; do
    echo compiling $file...
done
