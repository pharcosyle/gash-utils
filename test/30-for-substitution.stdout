compiling foo.o...
compiling bar.o...
